Solutions/0.2-Elliptic-Curve-Math-Solutions.ipynb

```diff
@@ -0,0 +1,272 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Add Path to Bitcoin Core source directory.\n",
```

Comment:
I lean towards recommending that we put the solutions to the exercises in text files instead of their own workbook. 

1.  we dont need to duplicate the workbook
2. we dont need a second workbook
3. students can just copy the text solution (or visually reference it as a hint) into their already-running workbook

thoughts?

Reply:
agree. Done